### ISSUE TITLE
Installing plugin doens't work and shows [object Filelist]

### ISSUE THREAD

**Issue by JVT038** [binhex/arch-delugevpn#264]
I'm using deluge 2.04dev38 and I tried to add [autoremoveplus 2.0.0](https://github.com/tote94/deluge-autoremoveplus), but it said 'object fileList' and it wasn't added.
How do I fix this? 
With kind regards,

**Comment by rwohleb** [comment]
You'll need to copy the egg file into the `plugins` directory inside of the appdata directory for your delugevpn install, and then restart the container.

**Comment by akanealw** [comment]
I'm having the same issue with the webapi plugin. I've moved the .egg file to the plugins directory in my mapped volume but it's not showing up when I restart Deluge.

**Comment by akanealw** [comment]
I was able to fix it. I was able to get a Python 3.8 WebAPI-0.4.0-py3.8.egg file from this issue - https://github.com/idlesign/deluge-webapi/issues/27. I stopped the container and copied over the .egg file into the /config/plugins directory. I renamed it just WebAPI.egg and added "WebAPI" to the core.conf file in the /config directory. Once I started up the container again, the WebAPI menu item showed up in the WebUI.
